mlflow/ml-package-versions.yml

```diff
@@ -312,7 +312,7 @@ spark:
 
   models:
     minimum: "3.0.0"
-    maximum: "3.1.2"
+    maximum: "3.2.0"
```

Comment:
We need to fix `set_matrix.py` to allow specifying unreleased versions:
https://github.com/mlflow/mlflow/blob/73c97db4adc87b7d8b57c243ec20a51ea7155162/dev/set_matrix.py#L171-L174

Reply:
Done! Added `allow_unreleased_max_version`